src/lib/markbind/src/parser.js

```diff
@@ -1,10 +1,11 @@
 const cheerio = require('cheerio');
 const fs = require('fs');
 const htmlparser = require('htmlparser2'); require('./patches/htmlparser2');
-const nunjucks = require('nunjucks');
 const path = require('path');
 const Promise = require('bluebird');
 const slugify = require('@sindresorhus/slugify');
+const nunjucks = require('nunjucks').configure()
```

Comment:
there's a few usages in `site.js` and `page.js` as well; Perhaps a common `nunjucksUtils` that does this and reexports to the three files could be used ( something like #1049 )

Reply:
We could do nunjucksUtils in a separate PR. For now I will repeat the `configure()` pattern across the different instances.

Reply:
I'm fine with this, #1049 has been merged though, depending on your conflict resolution goes might be nice to do it here instead 😁